src/KeyringController.ts

```diff
@@ -579,6 +586,64 @@ class KeyringController extends EventEmitter {
     return keyring.exportAccount(address, { withAppKeyOrigin: origin });
   }
 
+  /**
+   * Convert a base transaction to a base UserOperation.
+   *
+   * @param rawAddress - Address of the sender.
+   * @param transactions - Base transactions to include in the UserOperation.
+   * @returns A pseudo-UserOperation that can be used to construct a real.
+   */
+  async prepareUserOperation(
+    rawAddress: string,
+    transactions: EthBaseTransaction[],
+  ): Promise<EthBaseUserOperation> {
+    const address = normalizeToHex(rawAddress) as Hex;
```

Comment:
Isn't `normalizeToHex ` already returning a `Hex`? Not sure why the type casting is required here and the below methods

Reply:
Actually `normalizeToHex` has the following signature:

```ts
function normalize(input: number | string): string | undefined;
```

I kept the same pattern used in the other signing methods above, but I agree that it's not ideal. We should probably update the `normalize` method from `@metamask/eth-sig-utils` to always return a `Hex`.